agents/craftassist/craftassist_agent.py

```diff
@@ -499,7 +500,7 @@ def add_self_memory_node(self):
             p = self.get_player()
         except:  # this is for test/test_agent
             return
-        PlayerNode.create(self.memory, p, memid=self.memory.self_memid)
```

Comment:
Quick question - since we already create a PlayerNode/SelfNode in `sql_memory.py` why do we need to also create one here ?

Reply:
we don't- instead we just do the update now to make sure the agent's position and look are stored (they are Null when the SelfNode is created in memory because the agent isn't fully instantiated yet)